package.json

```diff
@@ -64,6 +64,8 @@
     "jstz": "~1.0.7",
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
+    "patternfly": "^3.29.5",
+    "patternfly-react": "^0.8.0",
```

Comment:
if you bump to 0.8.1 you'll get the int to String casting and can remove your custom code.